postgraphile/postgraphile/src/cli.ts

```diff
@@ -186,16 +185,10 @@ export async function run(args: ArgsFromOptions<typeof options>) {
     }
     const schemas = rawSchema?.split(",") ?? ["public"];
     const adaptor =
-      preset.pgServices?.[0]?.adaptor ?? "@dataplan/pg/adaptors/pg";
+      preset.pgServices?.[0]?.adaptor ??
+      (await import("@dataplan/pg/adaptors/pg"));
 
-    const importSpecifier = adaptor.match(/^([a-z]:|\.\/|\/)/i)
-      ? pathToFileURL(adaptor).href
-      : adaptor;
-
-    const mod = await import(importSpecifier);
-    const makePgService = (mod.makePgService ?? mod.default?.makePgService) as (
-      options: MakePgServiceOptions,
-    ) => GraphileConfig.PgServiceConfiguration;
+    const makePgService = adaptor.makePgService;
     if (typeof makePgService !== "function") {
```

Comment:
Please re-add the ESM shenanigans avoidance code, otherwise my support burden will increase :sweat_smile: 

```ts
let tmp;
const adaptor =
      preset.pgServices?.[0]?.adaptor ??
      ((tmp = await import("@dataplan/pg/adaptors/pg")) && tmp.makePgService ? tmp : tmp.default);
```